server/graphql/v2/object/Policies.ts

```diff
@@ -133,5 +139,31 @@ export const GraphQLPolicies = new GraphQLObjectType({
         }
       },
     },
+    [POLICIES.CONTRIBUTOR_INFO_THRESHOLDS]: {
+      type: new GraphQLObjectType({
+        name: POLICIES.CONTRIBUTOR_INFO_THRESHOLDS,
+        fields: () => ({
+          legalName: { type: GraphQLInt },
+          address: { type: GraphQLInt },
+        }),
+      }),
+      description:
+        'Contribution threshold to enforce contributor info. This resolver can be called from the collective or the host, when resolved through the collective the thresholds are returned in the collective currency',
+      async resolve(account, args, req) {
+        const host =
+          account.HostCollectiveId && account.HostCollectiveId !== account.id
+            ? await req.loaders.Collective.byId.load(account.HostCollectiveId)
+            : account;
+        let thresholds = await getPolicy(host, POLICIES.CONTRIBUTOR_INFO_THRESHOLDS, req);
+        let fxRate = 1;
+        if (!thresholds || isEmpty(thresholds)) {
+          fxRate = await getFxRate('USD', account.currency);
+          thresholds = DEFAULT_USD_THRESHOLDS;
+        } else if (host.currency !== account.currency) {
+          fxRate = await getFxRate(host.currency, account.currency);
```

Comment:
:ice_cream: Here, I'd recommend using `req.loaders.CurrencyExchangeRate.fxRate`, as it will save some network requests if we ever start fetching this in a list.